src/index.html

```diff
@@ -2,11 +2,48 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar">
+      <section class="calendar__field calendar__field--start-day--sun calendar__field--month-length--29">
+        <p class="calendar__cell calendar__cell--day--1"></p>
```

Comment:
don't create separate class with day number for each day
use nth-child in css for selecting needed day